src/apis/treasury.ts

```diff
@@ -24,4 +37,37 @@ export class DefaultTreasuryAPI implements TreasuryAPI {
         const account = await this.api.query.polkaBtc.account(id);
         return account.free;
     }
+
+    /**
+     * @param events The EventRecord array returned after sending transfer transaction
+     * @returns A boolean value
+     */
+    isTransferSuccessful(events: EventRecord[]): boolean {
+        for (const {
+            event: { method, section },
+        } of events) {
+            if (section == "polkaBtc" && method == "Transfer") {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * @param destination The address of a user
+     * @param amount The Amount for a transfer
```

Comment:
please rephrase: `@param amount - the amount in satoshi to transfer`

Reply:
Also renaming the parameter to `amountSatoshi` would be nice